### ISSUE TITLE
[4.1] Activities on spells cannot be set to *not* expend a spell slot

### ISSUE THREAD

**Issue by krbz999** [foundryvtt/dnd5e#4632]
![image](https://github.com/user-attachments/assets/97cfaee1-b64d-451f-8bc5-3558df289ae4)

**Comment by krbz999** [comment]
Also, as pictured, cannot scale with number of uses.

**Comment by arbron** [comment]
Doesn't seem to be the case. If you have the spell set to Prepared mode and "Consume Spell Slot" is unchecked it will still by used fine, but you will get the warning if you don't have any spellcasting slots at all because scaling is always based on a spell slots.

**Comment by nschoenwald** [comment]
> Doesn't seem to be the case. If you have the spell set to Prepared mode and "Consume Spell Slot" is unchecked it will still by used fine, but you will get the warning if you don't have any spellcasting slots at all because scaling is always based on a spell slots.

I don't think you should get a warning when casting a spell without spell slots if you don't have spell slots at the moment. After all, you don't need to have one. That seems counter-intuitive and likely to trip players up.
